docs/contributors/release.md

```diff
@@ -253,6 +253,128 @@ Now, the branch is ready to be used to publish the npm packages.
 
+
+### Standalone Package Releases 
+
+The following workflow is needed when packages require bug fixes or security releases to be published to _npm_ outside of a WordPress release cycle.
+
+Note: Both the `master` and `wp/trunk` branches are restricted and can only be _pushed_ to by the Gutenberg Core team.
+
+Identify the commit hashes from the pull requests that need to be ported from the repo `master` branch to `wp/trunk`
+
+The `wp/trunk` branch now needs to be prepared to release and publish the packages to _npm_.
+
+Open a terminal and perform the following steps:
+1. `git checkout master`
+2. `git pull`
+3. `git checkout wp/trunk`
+4. `git pull`
+
+Before porting commits check that the `wp/trunk` branch does not have any outstanding packages waiting to be published:
+1. `git checkout wp/trunk`
+2. `npm run publish:check`
+
+Now _cherry-pick_ the commits from `master` to `wp/trunk`, use `-m 1 commithash` if the commit was a pull request merge commit:
+1. `git cherry-pick -m 1 cb150a2`
+2. `git push` 
+
+Whilst waiting for the Travis CI build for  `wp/trunk` [branch to pass](https://travis-ci.com/WordPress/gutenberg/branches) identify and begin updating the `CHANGELOG.md` files:
+1. `git checkout wp/trunk`
+2. `npm run publish:check`
+> Example
+> ```shell
+> npm run publish:check
+> @wordpress/e2e-tests
+> @wordpress/jest-preset-default
+> @wordpress/scripts
+> lerna success found 3 packages ready to publish
+> ```
+
+Check the versions listed in the current `CHANGELOG.md` file, looking through the commit history of a package e.g [@wordpress/scripts](https://github.com/WordPress/gutenberg/commits/master/packages/scripts) and look out for _"chore(release): publish"_ and _"Update changelogs"_ commits to determine recent version bumps, then looking at the commits since the most recent release should aid with discovering what changes have occurred since the last release.
+
+Note: You may discover the current version of each package is not up to date, if so updating the previous released versions would be appreciated.
+
+Begin updating the _changelogs_ based on the [Maintaining Changelogs](https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs) documentation and commit the changes:
+
+1. `git checkout wp/trunk`
+2. Update each of the `CHANGELOG.md` files
+3. Stage the _changelog_ changes `git add packages/`
+4. `git commit -m "Update changelogs"`
+5. Make a note of the commit hash of this commit
+> Example
+> ```
+> [master 278f524f16] Update changelogs` 278f524
+> ```
+6. `git push`
+
+Now that the changes have been committed to the `wp/trunk` branch and the Travis CI builds for the `wp/trunk` [branch are passing](https://travis-ci.com/WordPress/gutenberg/branches) it's time to publish the packages to npm:
+1. Once again run `npm run publish:check` to confirm there are no unexpected packages ready to be published:
```

Comment:
I don't think this step is necessary at this point. CHANGELOG files are ignored by Lerna.

Reply:
Agreed, but whilst cherry-picking between `master` and `wp/trunk` it's a nice safeguard to ensure you haven't cherry-picked and incorrect commit that would result in another package up for release 😏

Personally, as I was doing just that I wanted to make sure as I did each of the steps I documented the steps, I also did this over the course of ~2 days, an hour here, an hour there, so re-running these steps `git pull`, `git status`, `npm run publish:check` ensured I had not made a mistake 😬